### ISSUE TITLE
DLL conflict with msys64 mingw64 libpangoft2-1.0-0.dll

### ISSUE THREAD

**Issue by exrom** [doublecmd/doublecmd#752]
### Version

doublecmd-1.0.9.x86_64-win64.msi
OS: Win10 21H2 (Build 19044.2486)

### Description

Hello,

i face a popup message every time i open DC. I installed doublecmd-1.0.9.x86_64-win64.msi on my x64 Win10.
The popup says:
Der Prozedureinsprungpunkt "hb_font_set_variantions" wurde in der DLL "C:\msys64\mingw64\bin\libpangoft2-1.0-0.dll" nicht gefunden
It's german unfortunately but probably means "procedure entry point ... could not be located in the dynamic link library..."

The problem does not occur when i install doublecmd-1.0.9.i386-win32.msi instead.
The problem also does not occur when i empty PATH envvar before starting 
"c:\Program Files\Double Commander\doublecmd.exe"

TCMD's Info says about this DLL c:\msys64\mingw32\bin\libpangoft2-1.0-0.dll:
CompanyName: "Red Hat Software"
FileDescription: PangoFT2
FileVersion: 1.43.0.0
InternalName: pangoft2-1.0-0
LegalCopyright: "Copyright 1999 Red Hat Software. Copyright 2000 Tor Lillqvist"
OriginalFilename: pangoft2-1.0-0.dll
ProductName: PangoFT2
ProductVersion: 1.43.0
Language: "Englisch (Vereinigte Staaten)"
File Version: 1.43.0.0
Product Version: 1.43.0.0
File OS: "32-bit Windows"
File Type: DLL
File Modified: "30.07.2019 09:09:08"
File Created: "31.12.2019 10:29:12"
File Size: "97.378 bytes"
Entry Point Raw Offset: 0x00000760
Compiler: Unknown
Compiler (PEiD): Unknown

would be great if you could provide a bug fix or a workaround.
I just came across DC as a cross platform alternative to tcmd and it seems the perfect choice. Thank you!

**Comment by alexx2000** [comment]
Hi, I guess you have two different libpangoft2 in the %PATH% environment variable, what causes version conflicts. Remove one of them from %PATH%.

**Comment by exrom** [comment]
Hi alexx2000, 
thanks for the answer.
I indeed found another dll with similar name
>where * | findstr pangof
C:\Program Files\Graphviz 2.43.20200408.0903\bin\pangoft2-1.dll
C:\msys64\mingw64\bin\libpangoft2-1.0-0.dll

Removing the Graphviz dll does not fix the issue.
Removing the mingw64 libpangoft2-1.0-0.dll fixes the issue as well as removing it from PATH. But as i need this to be in PATH removing is not an option here.

Anyhow, i found another crazy workaround.
I created a folder (!) called "libpangoft2-1.0-0.dll" right beside the doublecmd.exe which fixes it :)
Seems, the current folder is the first one to be searched for the dll and the same named dummy will stop the search.
Did not observe any strange behaviour so far.
